internal/view/app.go

```diff
@@ -173,6 +172,49 @@ func (a *App) suggestCommand() model.SuggestionFunc {
 	}
 }
 
+func (a *App) suggestSubCommand(command string) []string {
+	cmds := strings.Split(command, " ")
+	if len(cmds[0]) == 0 || len(cmds) != 2 {
+		return nil
+	}
+
+	var suggests []string
+	switch strings.ToLower(cmds[0]) {
+	case "cow", "q", "q!", "qa", "Q", "quit", "?", "h", "help", "a", "alias", "x", "xray", "dir":
+		return nil // ignore special commands
+	case "ctx", "context", "contexts":
+		ctxs, err := a.factory.Client().Config().Contexts()
+		if err != nil {
+			log.Error().Err(err).Msg("failed to list contexts")
+			return nil
+		}
+
+		for contextName := range ctxs {
+			if suggest, ok := shouldAddSuggest(cmds[1], contextName); ok {
+				suggests = append(suggests, suggest)
+			}
+		}
+	default:
+		nss, err := a.factory.Client().ValidNamespaces()
```

Comment:
wouldn't make sense to limit only at the search bar and within the given namespace?
I would be annoyed if I moved accidentally to a resources from a different namespace

Reply:
@placintaalexandru Thank you for weighting in! Could you clarify, don't think I understood your point??

Reply:
hello @derailed 
the original issue was to add search suggestions: for example if i am in the deployment view and type `my-deployment`, the search bar would suggest everything starting with `my-deployment` within the current namespace

Reply:
Thank you for the clarification @placintaalexandru! I see what you are saying now. The original issue which Jayson implemented here is correct. I believe this issue relates to completing on the namespace when users enter a command ie `dp xxx` where xxx denotes suggestions from matching namespaces. 
Best I can tell when the user activates a filter aka `/` we just want to filter the resources based on what the user types i.e no autocomplete. Does this make sense?